### ISSUE TITLE
Build failure with ghc 7.10 and aeson < 1.0

### ISSUE THREAD

**Issue by NCrashed** [GetShopTV/swagger2#71]
The problem is [here](https://github.com/GetShopTV/swagger2/blob/master/src/Data/Swagger/Internal/Schema.hs#L506):
```
Preprocessing library swagger2-2.1.2...
[ 1 of 14] Compiling Data.Swagger.Internal.AesonUtils ( src/Data/Swagger/Internal/AesonUtils.hs, dist/build/Data/Swagger/Internal/AesonUtils.o )
[ 2 of 14] Compiling Data.Swagger.SchemaOptions ( src/Data/Swagger/SchemaOptions.hs, dist/build/Data/Swagger/SchemaOptions.o )
[ 3 of 14] Compiling Data.Swagger.Internal.Utils ( src/Data/Swagger/Internal/Utils.hs, dist/build/Data/Swagger/Internal/Utils.o )
[ 4 of 14] Compiling Data.Swagger.Declare ( src/Data/Swagger/Declare.hs, dist/build/Data/Swagger/Declare.o )
[ 5 of 14] Compiling Data.Swagger.Internal ( src/Data/Swagger/Internal.hs, dist/build/Data/Swagger/Internal.o )
[ 6 of 14] Compiling Data.Swagger.Lens ( src/Data/Swagger/Lens.hs, dist/build/Data/Swagger/Lens.o )
[ 7 of 14] Compiling Data.Swagger.Internal.ParamSchema ( src/Data/Swagger/Internal/ParamSchema.hs, dist/build/Data/Swagger/Internal/ParamSchema.o )
[ 8 of 14] Compiling Data.Swagger.ParamSchema ( src/Data/Swagger/ParamSchema.hs, dist/build/Data/Swagger/ParamSchema.o )
[ 9 of 14] Compiling Data.Swagger.Internal.Schema ( src/Data/Swagger/Internal/Schema.hs, dist/build/Data/Swagger/Internal/Schema.o )

src/Data/Swagger/Internal/Schema.hs:506:3:
    parse error on input ‘where’
```

ghc: 7.10.3 
aeson: 0.11.2.1

**Comment by NCrashed** [comment]
Thanks! Could you publish fixed version to Hackage?

**Comment by phadej** [comment]
Fixed in `swagger2-2.1.2.1`

**Comment by NCrashed** [comment]
Thank you!
